system-x/services/graphql/src/main/java/software/tnb/splunk/resource/local/LocalGraphQL.java

```diff
@@ -0,0 +1,48 @@
+package software.tnb.splunk.resource.local;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.auto.service.AutoService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import software.tnb.common.deployment.Deployable;
+import software.tnb.splunk.service.GraphQL;
+
+@AutoService(GraphQL.class)
+public class LocalGraphQL extends GraphQL implements Deployable {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LocalGraphQL.class);
+
+    private GraphQLContainer container;
+
+    @Override
+    public void deploy() {
+        LOG.info("Starting GraphQL container");
+        List<Integer> ports = new ArrayList<>();
+        ports.add(PORT);
+        container = new GraphQLContainer(defaultImage(), ports);
```

Comment:
sorry, I forgot to mention that you should use `image()` here when implementing `WithDockerImage`

Reply:
Done - can you point me to docs or describe what is happening here ?